### DISCUSSION TITLE
HDF5 error: Object 'GeneScoreMatrix/chrX/x' does not exist in this HDF5 file.

### DISCUSSION THREAD

**Discussion by bj-chen** [GreenleafLab/ArchR#1285] [Questions / Documentation]
Hi ArchR team,

I was trying to subset 5k cells from the original ArchR object which contains ~260K cells. However, I run into problems with the function getMarkerFeatures which gives the following error:

Error in .safelapply(seq_along(matchObj[[1]]), function(x) { : 
Error Found Iteration 3 : 
	[1] "Error in h5read(ArrowFile, paste0(useMatrix, \"/\", seqnamex, \"/x\")) : \n  Object 'GeneScoreMatrix/chrX/x' does not exist in this HDF5 file.\n"
	<simpleError in h5read(ArrowFile, paste0(useMatrix, "/", seqnamex, "/x")): Object 'GeneScoreMatrix/chrX/x' does not exist in this HDF5 file.>
Error Found Iteration 4 : 
	[1] "Error in h5read(ArrowFile, paste0(useMatrix, \"/\", seqnamex, \"/x\")) : \n  Object 'GeneScoreMatrix/chrX/x' does not exist in this HDF5 file.\n"
	<simpleError in h5read(ArrowFile, paste0(useMatrix, "/", seqnamex, "/x")): Object 'GeneScoreMatrix/chrX/x' does not exist in this HDF5 file.>
Calls: <Anonymous> ... getMarkerFeatures -> do.call -> <Anonymous> -> .safelapply
Execution halted

I attached the logfile and would really appreciate any help.

Thank you!

[ArchR-getMarkerFeatures-5f6e43f99ed2-Date-2022-02-10_Time-12-15-00.log](https://github.com/GreenleafLab/ArchR/files/8038281/ArchR-getMarkerFeatures-5f6e43f99ed2-Date-2022-02-10_Time-12-15-00.log)

**Comment by rcorces** [comment]
Since this is an error report, it should go to Issues.
